README.md

```diff
@@ -72,3 +72,30 @@ For full details on how to use Streamlink visit our documentation at [streamlink
 Feel free to open a bug or contribute to code!
 
 No need to go through a large CONTRIBUTING.md doc. The only requirement being that we get at least one ACK from a maintainer. Fork / clone us and open a PR!
+
+<a href="graphs/contributors"><img src="https://opencollective.com/streamlink/contributors.svg?width=890" /></a>
```

Comment:
TBH, this also fits into "Contributing". Then we don't have to add a new section with just a big single image as content (which looks weird in combination with the header).

Are HTML tags necessary, though? It's a simple image that could be perfectly added via regular markdown and wouldn't break the style of the document (I'm talking about the images and links at the bottom as well).